content/en-us/scripting/events.md

```diff
@@ -77,7 +77,7 @@ local touchedPart = part.Touched:Wait()
 print("The part was touched by", touchedPart:GetFullName())
 ```
 
-The `Global.RobloxGlobals.wait()|Wait` function returns the event's arguments. You can assign these arguments to variables where `Global.RobloxGlobals.wait()|Wait` is called.
+The `Wait` function returns the event's arguments. You can assign these arguments to variables where `Wait` is called.
```

Comment:
Do you mean to use `task.wait()` here too?

```suggestion
The `Library.task.wait()` function returns the event's arguments. You can assign these arguments to variables where `Library.task.wait()` is called.
```

Reply:
Yup! i seems to forget this one too! sorry!

Reply:
The wait is not for `task.wait` but rather, RBXSignal:Wait()